src/components/navBar/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import { FaReact, FaBars } from 'react-icons/fa';
+import { Link } from 'react-router-dom';
+import { HiX } from 'react-icons/hi';
+import './styles.scss';
+
+const data = [
+  {
+    label: 'HOME',
+    to: '/',
+  },
+  {
+    label: 'ABOUT',
+    to: '/about',
+  },
+  {
+    label: 'PROJECTS',
+    to: '/projects',
+  },
+];
+
+const Navbar = () => {
+  const [toggleIcon, setToggleIcon] = useState(false);
+
+  const handleIcon = () => {
+    setToggleIcon(!toggleIcon);
+  };
+
+  return (
+    <div>
```

Comment:
For those HTML elements, make sure that do the best practice with accessibility and others.